simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/chargingstation/ManufacturerVariableAccessor.java

```diff
@@ -0,0 +1,67 @@
+package com.evbox.everon.ocpp.simulator.station.component.chargingstation;
+
+import com.evbox.everon.ocpp.common.CiString;
+import com.evbox.everon.ocpp.simulator.station.Station;
+import com.evbox.everon.ocpp.simulator.station.StationHardwareData;
+import com.evbox.everon.ocpp.simulator.station.component.variable.SetVariableValidator;
+import com.evbox.everon.ocpp.simulator.station.component.variable.VariableAccessor;
+import com.evbox.everon.ocpp.simulator.station.component.variable.VariableGetter;
+import com.evbox.everon.ocpp.simulator.station.component.variable.VariableSetter;
+import com.evbox.everon.ocpp.v20.message.centralserver.*;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class ManufacturerVariableAccessor extends VariableAccessor {
+
+    public static final String NAME = "Manufacturer";
+
+    private final Map<GetVariableDatum.AttributeType, VariableGetter> variableGetters = ImmutableMap.<GetVariableDatum.AttributeType, VariableGetter>builder()
+            .put(GetVariableDatum.AttributeType.ACTUAL, this::getActualValue)
+            .build();
+
+    private final Map<SetVariableDatum.AttributeType, SetVariableValidator> variableValidators = ImmutableMap.<SetVariableDatum.AttributeType, SetVariableValidator>builder()
+            .put(SetVariableDatum.AttributeType.ACTUAL, this::validateActualValue)
+            .build();
+
+    public ManufacturerVariableAccessor(Station station) {
+        super(station);
+    }
+
+    @Override
+    public String getVariableName() {
+        return NAME;
+    }
+
+    @Override
+    public Map<GetVariableDatum.AttributeType, VariableGetter> getVariableGetters() {
+        return variableGetters;
+    }
+
+    @Override
+    public Map<SetVariableDatum.AttributeType, VariableSetter> getVariableSetters() {
+        return Collections.emptyMap();
+    }
+
+    @Override
+    public Map<SetVariableDatum.AttributeType, SetVariableValidator> getVariableValidators() {
+        return variableValidators;
+    }
+
+    private SetVariableResult validateActualValue(Component component, Variable variable, SetVariableDatum.AttributeType attributeType, CiString.CiString1000 attributeValue) {
```

Comment:
No, it's going to be different everywhere. For now it might be the same at some places cause we don't provide real values and have some stubs. Although it could be generalized a bit more, I'd postpone it for later.